test/Release/CreateReleaseTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * @author: Ryan Wood <ryanw@clearbooks.co.uk>
+ * @created: 15/07/15
+ */
+
+namespace Clearbooks\Labs\Release;
+
+
+use Clearbooks\Labs\Release\Gateway\InMemoryReleaseGateway;
+use Clearbooks\Labs\Release\Gateway\ReleaseGateway;
+use Clearbooks\Labs\Release\CreateRelease\BlankRequestStub;
+use Clearbooks\Labs\Release\CreateRelease\StaticRequestStub;
+use Clearbooks\Labs\Release\UseCase\CreateRelease\Request;
+
+class CreateReleaseTest extends \PHPUnit_Framework_TestCase
+{
+
+    /**
+     * @var ReleaseGateway
+     */
+    private $releaseGateway;
+
+    /**
+     * @var CreateRelease
+     */
+    private $createRelease;
+
+    /**
+     * Sets up the fixture, for example, open a network connection.
+     * This method is called before a test is executed.
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->releaseGateway = new InMemoryReleaseGateway();
+        $this->createRelease = new CreateRelease( $this->releaseGateway );
+    }
+
+
+    /**
+     * @test
+     */
+    public function givenEmptyRequest_ReturnsFalse()
+    {
+        $response =  $this->createRelease( new BlankRequestStub() );
+        $this->assertFalse( $response->isSuccessful() );
+        $this->assertCount( 1, $response->getValidationErrors() );
```

Comment:
What's are the errors? 

I'm not overly convinced that a systematic approach to degenerate tests has been used.